.github/workflows/readmeupdate.yaml

```diff
@@ -0,0 +1,23 @@
+name: Add contributors
+on:
+  # schedule:
+  #   - cron:  '20 20 * * *'
+  push:
+    branches:
+      - master
+
+jobs:
+  add-contributors:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: 0b51d14n217/Annual-Yearbook@readme-workflow
+      with:
+        CONTRIBUTOR: 'https://github.com/ComputerSocietyVITC/Annual-Yearbook/graphs/contributors'
+        COLUMN_PER_ROW: '6'
+        ACCESS_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        IMG_WIDTH: '100'
+        FONT_SIZE: '14'
+        PATH: '/README.md'
+        COMMIT_MESSAGE: 'docs(README): update contributors'
+        AVATAR_SHAPE: 'round'
```

Comment:
These are data points, we should rather focus on automating it! The COMMIT_MESSAGE is good though